Code/Tools/ProjectManager/Source/ExternalLinkDialog.cpp

```diff
@@ -85,6 +85,7 @@ namespace O3DE::ProjectManager
 
     void ExternalLinkDialog::SetSkipDialogSetting(bool state)
     {
-        PMSettings::SetProjectManagerKey(PMSettings::GetExternalLinkWarningKey(), state);
+        auto settings = SettingsInterface::Get();
+        settings->Set(QString(settings->ExternalLinkWarningKey), state);
```

Comment:
can this be simplified to
```suggestion
        SettingsInterface::Get()->Set(ISettings::ExternalLinkWarningKey, state);
```

Also, I wonder if there is a way we could declare a static function called Set that would automatically handle the ::Get() for us so we end up with something like this:
```
SettingsInterface::Set(ISettings::ExternalLinkWarningKey, state);
```
or
```
ISettings::Set(ISettings::ExternalLinkWarningKey, state);
```

Reply:
The simplifications makes more sense. I'm not sure about automatically handling the ::Get(), I think its fine the way it is right now as that would just end up as a lot more boilerplate that just reduces the number of characters a bit to make a single call. At least for now it doesn't seem worth it.